### DISCUSSION TITLE
Github Issue Guide | /issue/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1353] [Comments]
https://yihui.org/issue/

[a comment by an automated account is not shown]

**Comment by salmankhan302** [comment]
Hello every on whne I try to kint Markdown file I got following error. Please help me to fix it 

processing file: HelloWorld.Rmd
                                                                                                            
"C:/Program Files/RStudio/resources/app/bin/quarto/bin/tools/pandoc" +RTS -K512m -RTS HelloWorld.knit.md --to latex --from markdown+autolink_bare_uris+tex_math_single_backslash --output HelloWorld.tex --lua-filter "C:\Users\tksasa\AppData\Local\R\win-library\4.4\rmarkdown\rmarkdown\lua\pagebreak.lua" --lua-filter "C:\Users\tksasa\AppData\Local\R\win-library\4.4\rmarkdown\rmarkdown\lua\latex-div.lua" --embed-resources --standalone --highlight-style tango --pdf-engine pdflatex --variable graphics --variable "geometry:margin=1in" 
output file: HelloWorld.knit.md

Error: LaTeX failed to compile HelloWorld.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips.
In addition: Warning messages:
1: In system2(..., stdout = if (use_file_stdout()) f1 else FALSE, stderr = f2) :
  'CreateProcess' failed to run 'C:\Users\tksasa\AppData\Roaming\TinyTeX\bin\windows\pdflatex.exe -halt-on-error -interaction=batchmode "HelloWorld.tex"'
2: In system2(...) :
  'CreateProcess' failed to run 'C:\Users\tksasa\AppData\Roaming\TinyTeX\bin\windows\pdflatex.exe -halt-on-error -interaction=batchmode "HelloWorld.tex"'
Execution halted
